js/foundation.reveal.js

```diff
@@ -469,6 +474,10 @@ class Reveal extends Plugin {
     this.$element.hide().off();
     this.$anchor.off('.zf');
     $(window).off(`.zf.reveal:${this.id}`);
+
+    if ($('.reveal:visible').length  === 0) {
```

Comment:
A comment would be great. Also, do you think the test should be duplicated or moved to `removeRevealOpenClasses` ?

Reply:
Not sure, the other function also just adds the class and does no prior check if is is already added.

Reply:
If I remember correctly `.addClass()` already do that check.

Reply:
Meant the check for `:visible`.

Reply:
Meant the `:visible` check around the calls.

Reply:
We may not have understood each other.

What I'm trying to say is that:
* For now, `addRevealOpenClasses` can be called on all Reveal initializations and will only add the class once (thanks to jQuery `.addClass`). So it takes care of the "check" to put the class only when needed.
* Maybe we should have a similar behavior with `removeRevealOpenClasses`: being callable on all Reveal destructions and take care of the "check" to only remove the class when needed. So with [the check](https://github.com/zurb/foundation-sites/pull/11041/files#diff-5b5e0861de0a3864db2c3c28919700b7L411) inside the function.

But it's a _maybe_, and I don't want to loose too much time on a so small thing 😄